data/mods/ssb/moves.ts

```diff
@@ -2899,6 +2899,42 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Ghost",
 	},
 
+	// nui
+	wincondition: {
+		accuracy: 100,
+		basePower: 0,
+		category: "Status",
+		desc: "Inflicts the opponent with random status of sleep, paralysis, burn, or toxic. Then uses Dream Eater, Iron Head, Fire Blast, or Venoshock, respectively.",
+		shortDesc: "Causes sleep and uses Dream Eater, para and Iron Head, burn and Fire Blast, or tox and Venoshock.",
+		name: "Win Condition",
+		isNonstandard: "Custom",
+		pp: 10,
+		priority: 0,
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, "Celebrate", target);
+		},
+		onHit(target, source) {
+			const hax = ['slp', 'brn', 'par', 'tox'];
+			target.trySetStatus(this.sample(hax), source);
```

Comment:
make this.sample(hax) its own variable for the if chain below

Reply:
Fine with me, but what if the target is already statused? E.g. nui used Win Condition the turn prior and burned the foe. Then using WinCondition this turn will fail a new status but should default to Fire Blast as the attack

Reply:
shouldnt matter. It would just attempt to status then use the corresponding move

Reply:
ok that makes it less broken